include/freerdp/peer.h

```diff
@@ -147,7 +148,13 @@ struct rdp_freerdp_peer
 	ALIGN64 psPeerGetEventHandles GetEventHandles;
 	ALIGN64 psPeerAdjustMonitorsLayout AdjustMonitorsLayout;
 	ALIGN64 psPeerClientCapabilities ClientCapabilities;
-	ALIGN64 psSspiComputeNtlmHash ComputeNtlmHash;
+#if defined(WITH_FREERDP_DEPRECATED)
+	WINPR_DEPRECATED_VAR("Use freerdp_peer::SspiNtlmHashCallback instead",
+	                     ALIGN64 psPeerComputeNtlmHash ComputeNtlmHash;)
+#else
+	UINT64 reserved2;
```

Comment:
If you are concerned about API/ABI compatility, then you need to add the new callback at the end of the struct, otherwise this `reserved2` field is useless.